web/tests.py

```diff
@@ -103,3 +104,15 @@ def test_coaches_page(self):
     def test_http404(self):
         response = self.client.get('/web/doesnotexist')
         self.assertEqual(response.status_code, 404)
+
+
+class RecordsPageTestCase(TestCase):
+
+    def test_uses_correct_template(self):
+        url = reverse('records')
+        response = self.client.get(url)
+        self.assertTemplateUsed(response, 'web/records.html')
+
+    def test_records_page(self):
```

Comment:
The `assertTemplateUsed` function in Django is a context manager. You can combine both tests like the following:

```python
def test_records_page(self):
    with self.assertTemplateUsed('web/records.html'):
        response = self.client.get('/web/records/')
        self.assertEqual(response.status_code, 200)
```